src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,51 @@
+import { PeopleFilters } from '../PeopleFilters/PeopleFilters';
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+import { useEffect, useState } from 'react';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [error, setError] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getPeople()
+      .then(setPeople)
+      .catch(() => setError('Something went wrong'))
```

Comment:
got a message from a mentor that this error text is not informative

Reply:
Good point, try to make up more informative message